test/system/system-test.ts

```diff
@@ -0,0 +1,27 @@
+/*
+	File: test/system/system-test.ts
+	cpuabuse.com
+*/
+
+/**
+ * Performs a series of system tests.
+ */
+import { spawn, fork, exec } from "child_process";
+
+/**
+ * All of the system tests.
+ */
+
+export async function systemTest(): Promise<void> {
+	await new Promise(function(resolve) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['resolve' is defined but never used.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5058624)